index.html

```diff
@@ -2,295 +2,392 @@
 layout: default
 ---
 
-<!-- Hero banner -->
-<section class="row hero-row">
-
-    <div class="wrapper">
-
-        <div class="eight-col">
-
-            <div class="logo">
-                <img class="logo__img" src="https://assets.ubuntu.com/v1/4fd55d51-picto-corepower-orange.svg" alt="Cloud Init logo" />
-                <span class="logo__txt">cloud-init</span>
-            </div>
-
-            <h1 class="hero-row__heading">The standard for customising cloud instances</h1>
-        </div>
-
-        <div class="six-col">
-            <a href="https://www.youtube.com/watch?v=-zL3BdbKyGY" class="hero__cta button--primary mediabox">
-                Watch tutorial
-            </a>
+<section class="p-strip p-strip--accent is-dark">
+  <div class="row">
+    <div class="col-12 suffix-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img"
+            src="https://assets.ubuntu.com/v1/4fd55d51-picto-corepower-orange.svg"
+            alt="">
+          <h3 class="p-heading-icon__title">cloud-init</h3>
         </div>
+      </div>
+      <h1>The standard for customising cloud instances</h1>
+      <a class="p-button--brand"
```

Comment:
Lets give this an ```aria-label``` explaining its going to youtube to watch a video, and what the content is about.